supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from conftest import Fields, TestFields
+from django.core.exceptions import FieldDoesNotExist, ValidationError
+from supplier.models import Supplier
+
+
+class TestSupplierModel:
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier0):
+        """Test - checks if supplier is saved in DB.
+
+        Args:
+            supplier0: fixture from module conftest.py.
+
+        Returns:
+            None.
+        """
+        Supplier.save_supplier(supplier0)
+        assert supplier0 in Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_supplier(self, saved_supplier0):
+        """Test - checks if supplier is not in DB after deletion.
+
+        Args:
+            saved_supplier0: fixture from module conftest.py.
+
+        Returns:
+            None.
+        """
+        Supplier.delete_supplier(saved_supplier0)
+        assert saved_supplier0 not in Supplier.objects.all()
```

Comment:
How do you know it was there before the test?

Reply:
delete supplier method takes care of that by throwing an "ObjectNotFound" error.
thou this feature hasn't been tested so good call.

Reply:
Make seperated functions for not exist and exist, just check if supplier exist before deleting since you save the supplier in the fixture

Reply:
Make seperated functions for not exist and exist, just check if supplier exist before deleting since you save the supplier in the fixture